Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -95,11 +95,11 @@
 #define LPMT_DEFAULT_NO_BUTTON_TEXT @"No"
 #define LPMT_DEFAULT_LATER_BUTTON_TEXT @"Maybe Later"
 #define LPMT_DEFAULT_URL @"http://www.example.com"
-#define LPMT_DEFAULT_CLOSE_URL @"http://leanplum:close"
-#define LPMT_DEFAULT_OPEN_URL @"http://leanplum:loadFinished"
-#define LPMT_DEFAULT_TRACK_URL @"http://leanplum:track"
-#define LPMT_DEFAULT_ACTION_URL @"http://leanplum:runAction"
-#define LPMT_DEFAULT_TRACK_ACTION_URL @"http://leanplum:runTrackedAction"
+#define LPMT_DEFAULT_CLOSE_URL @"http://leanplum/close"
```

Comment:
Can you pull the base URL out into it's own value?
LPM_DEFAULT_BASE_URL = "http://leanplum/"
LPMT_DEFAULT_CLOSE_URL = DEFAULT_BASE_URL + "close";